File: leetcode/0263.go
package leetcode

func isUgly(n int) bool {
	if n < 1 {
		return false
	}
	var nums = [3]int{2, 3, 5}
	for _, num := range nums {
		for n%num == 0 {
			n /= num
		}
	}
	return n == 1
}

File: leetcode/0263.cpp
class Solution {
public:
    bool isUgly(int n) {
        if (n < 1) return false;
        int nums[3]{2, 3, 5};
        for (auto i : nums)
            while (n % i == 0) n /= i;
        return n == 1;
    }
};